OJP/OJP_Trips.xsd

```diff
@@ -743,6 +743,38 @@
 					<xs:documentation>Number how often the access feature occurs in this PathLink</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="From" type="PathLinkEndStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Designations of the level where the PathLink starts.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="To" type="PathLinkEndStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Designations of the level where the PathLink ends.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="PathLinkEndStructure">
+		<xs:annotation>
+			<xs:documentation>Designations of a floor.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="LevelPublicCode" type="xs:normalizedString" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Public identifier of the level as found on elevators and signs.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="LevelName" type="InternationalTextStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Official name of the level.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Id" type="xs:normalizedString" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Id of the element at the From or To end of the PathLink (typically a PLACE).</xs:documentation>
```

Comment:
Id of the element at this end of the PathLink (typically a PLACE, e.g. elevator)

Reply:
@sgrossberndt OK - but perhaps without 'e.g. elevator'? I think the Id would typically point at the EquipmentPlace of the elevator (which has coordinates), not directly at the LiftEquipment. -> or: 'e.g., where the elevator is located' ?

Reply:
integrated your discussion